src/pages/index.js

```diff
@@ -29,14 +29,14 @@ const Home = () => (
       <ResourceCard
         lead="Experience VR"
         title="Firefox"
-        desc="Experience WebVR with Firefox; Mozilla's fast, private and ethical web browser."
+        desc="Experience WebVR with Firefox: Mozilla's fast, private, and ethical web browser."
         link="https://www.mozilla.org/firefox/new/"
         linkText="Get it Now"
       />
       <ResourceCard
         lead="Build with Unity"
         title="Unity WebVR Assets"
-        desc="Publish and share VR experiences created in Unity on the open web."
+        desc="Publish and share VR experiences created in Unity on the open Web."
```

Comment:
["open Web"](https://github.com/MozillaReality/vr-mozilla-org/pull/34/files#diff-84099094c0c18f11b3c43fe0e8959b97R19) is capitalised above. let me know whether you want it capitalised or not everywhere (also see [above](https://github.com/MozillaReality/vr-mozilla-org/pull/34/files#diff-84099094c0c18f11b3c43fe0e8959b97R32): "ethical web browser").